arrays/arrays.c

```diff
@@ -103,9 +113,12 @@ void arr_append(Array *arr, char *element) {
   // or throw an error if resize isn't implemented yet.
 
   // Copy the element and add it to the end of the array
+  arr->elements[arr->count] = element;
```

Comment:
You should make a copy of 'element' and then assign the elements[count] to this copy.  Brady will go over the reason later.